src/modules/rubberband/filter_rbpitch.cpp

```diff
@@ -38,8 +38,6 @@ typedef struct
     uint64_t out_samples;
 } private_data;
 
-static const size_t MAX_CHANNELS = 10;
```

Comment:
I checked the rubberband library and it does not not have a limit on the channel count. I set this to 10 initially because that is the highest number of channels that I ever tested.